parser/fileparser.go

```diff
@@ -103,22 +103,23 @@ func scanTSVHeader(fileScanner *bufio.Scanner) (*fpt.BroHeader, error) {
 		//On the comment lines
 		if fileScanner.Text()[0] == '#' {
 			line := strings.Fields(fileScanner.Text())
-			if strings.Contains(line[0], "separator") {
-				//TODO: Figure out how to escape read in string
-				if line[1] == "\\x09" {
-					toReturn.Separator = "\x09"
+			if line[0][1:] == "separator" {
```

Comment:
Maybe consider collapsing this whole tree into a switch statement.  Might clean it up a little bit but that might just be personal preference too.

Reply:
cleaned up